README.md

```diff
@@ -10,7 +24,7 @@ The documentation can be found on http://ipython-extensions.readthedocs.org/en/l
 
 You have an IPython extension (e.g. a `%magic` command) but it's too small to build your own package? You are welcome to contribute it to this repository!
 
-To prevent bit rot, it should follow the guidelines outlined in [CONTRIBUTING.md](CONTRIBUTING.md), meaning:
+To prevent bit rot, it should follow the guidelines outlined in [CONTRIBUTING.md](https://github.com/ipython-contrib/IPython-extensions/blob/master/CONTRIBUTING.md), meaning:
```

Comment:
This is needed for PyPi, which otherwise displays a relative link which of course does not work :-(